design/data_only_restore.md

```diff
@@ -0,0 +1,141 @@
+# Data-only restore design
+
+## Abstract
+Support to restore only the data in the backup volume without restoring the k8s resources.
+
+## Background
+In some disaster recovery scenarios, the user only wants to restore the data in the volumes, because only specific volume's data is corrupted, but Velero doesn't support that yet. The design aims to address that missing function.
+
+## Goals
+- Support to restore only volume data in the backup.
+- Can filter the volumes that needs data restore.
+
+## Non Goals
+- Will not support the in-place volume data restore. To achieve data integrity, new PVC and PV are created, and wait for the user to mount the restored volume.
```

Comment:
Heard from US community meeting that this will be moved into Goals.

Would have to discuss how we're handling volumes that are already mounted.. or do we say, you have to pre-delete/scaledown so volumes are not mounted by restore time.